### PR TITLE
Solution-Pack-USB-Management-Exceptions

### PR DESCRIPTION
**Pull request by IslamkamalBaker**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,33 @@
-# solution-template-repository
\ No newline at end of file
+## About the connector
+Fortisoar powerful tool to automate manual and repetitive tasks, especially when it comes to security and infrastructure? Juggling USB exception requests can be a real pain, with endless emails, approvals, and documentation. But what if there was a better way?
```
Comment (cs-rohit-khune) [inline README.md:10]: Please follow the standard format for `README.md`. Refer to [this](https://github.com/fortinet-fortisoar/solution-pack-phishing-email-response/blob/develop/README.md)

File: connectors/data.json [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    {
+        "name": "activedirectory",
+        "label": "Active Directory",
+        "category": [
+            "Directory Service"
+        ],
+        "description": "Active Directory (AD) is a directory service that Microsoft developed for Windows domain networks. You can directly query AD to retrieve information about users, groups, and computers, in an organization, by using the Lightweight Directory Access Protocol (LDAP) to directly query the AD.",
+        "publisher": "Fortinet",
+        "operation_roles": [],
+        "configurations": [
```
Comment (cs-rohit-khune) [inline connectors/data.json:11]: Please remove the connector configuration

File: connectors/data.json [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+        ],
+        "description": "The Exchange connector provides a robust, platform-independent, and simple interface for communicating with Microsoft Exchange 2007-2016 Server or Office 365 using Exchange Web Services (EWS).",
+        "publisher": "Fortinet",
+        "operation_roles": [],
+        "configurations": [],
+        "dataImports": [],
+        "install_mode": "rpm"
+    },
+    {
+        "name": "fortinet-fortiedr_dev",
```
Comment (cs-rohit-khune) [inline connectors/data.json]: Use to the production version of the connector, replacing `fortinet-fortiedr_dev` with `fortinet-fortiedr`.

File: connectors/data.json [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+        "category": [
+            "Endpoint Protection"
+        ],
+        "description": "FortiEDR protects endpoints pre and post infection, stopping data breaches in real-time and automatically orchestrating incident investigation and response. This connector facilitates the automated operations related to events, forensics and collectors.",
+        "publisher": null,
+        "version": "2.0.0",
+        "operation_roles": [],
+        "configurations": [
+            {
+                "id": 111,
```
Comment (cs-rohit-khune) [inline connectors/data.json:66]: Please remove the connector configuration

File: info.json [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+    "publisher": "Islam Baker",
+    "description": "Fortisoar powerful tool to automate manual and repetitive tasks, especially when it comes to security and infrastructure? Juggling USB exception requests can be a real pain, with endless emails, approvals, and documentation. But what if there was a better way?\n\nIn this Solution pack, we'll show you exactly how Fortisoar can:\n\n\u2022\tEliminate tedious information gathering: Say goodbye to chasing down user details and approval statuses. Fortisoar captures everything automatically, saving you precious time and frustration.\n\u2022\tSimplify approvals: No more manual back-and-forth with managers. Fortisoar automates the approval process, ensuring it's efficient and transparent.\n\u2022\tAutomate access expiration: Never worry about forgetting to remove permissions again. Fortisoar automatically sets and enforces expiration dates, keeping your systems secure.\n\u2022\tStreamline documentation: Ditch the paper trail! Fortisoar keeps a complete audit log of all requests and approvals, readily available for compliance purposes.\n\nFortisoar can transform your USB exception management",
+    "help": null,
```
Comment (cs-rohit-khune) [inline info.json:19]: Add README.md link here - https://github.com/fortinet-fortisoar/solution-pack-usb-management-exception/blob/release/1.0.0/README.md

File: playbooks/USB Approval system/collection.metadata.json [hunk 6]
```diff
@@ -0,0 +1,13 @@
+{
+    "@context": "\/api\/3\/contexts\/WorkflowCollection",
+    "@type": "WorkflowCollection",
+    "name": "USB Approval system",
```
Comment (cs-rohit-khune) [inline playbooks/USB Approval system/collection.metadata.json]: Rename the playbook collection to "02 - Use Case - USB Approval System" to follow the standard format.

File: info.json [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+                "name": "Active Directory",
+                "apiName": "activedirectory"
+            },
+            {
+                "name": "Exchange",
+                "apiName": "exchange"
+            },
+            {
+                "name": "Fortinet FortiEDR",
+                "apiName": "fortinet-fortiedr_dev",
```
Comment (cs-rohit-khune) [inline info.json]: Use to the production version of the connector, replacing `fortinet-fortiedr_dev` with `fortinet-fortiedr`.

File: info.json [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+                "version": "2.0.0"
```
Comment (cs-rohit-khune) [inline info.json]: Remove the version key

**Review by cs-rohit-khune — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (IslamkamalBaker) [reply README.md:10]: Done

File: connectors/data.json (hunk 2 above)
Comment (IslamkamalBaker) [reply connectors/data.json:11]: Done

File: connectors/data.json (hunk 3 above)
Comment (IslamkamalBaker) [reply connectors/data.json]: Done

File: connectors/data.json (hunk 4 above)
Comment (IslamkamalBaker) [reply connectors/data.json:66]: Done

File: info.json (hunk 5 above)
Comment (IslamkamalBaker) [reply info.json:19]: i need help on that

File: playbooks/USB Approval system/collection.metadata.json (hunk 6 above)
Comment (IslamkamalBaker) [reply playbooks/USB Approval system/collection.metadata.json]: Done

File: info.json (hunk 7 above)
Comment (IslamkamalBaker) [reply info.json]: Done

File: info.json (hunk 8 above)
Comment (IslamkamalBaker) [reply info.json]: Done

**Pull request merged**